### PR TITLE
[FLINK-7560] Add reset button for plan visualizer page

### PR DESCRIPTION
**Pull request by yew1eb**
[#Issue FLINK-7560](https://issues.apache.org/jira/browse/FLINK-7560)

### CODE REVIEW

**Comment by yew1eb** [comment]
wx20170829-235929 2x

**Comment by yew1eb** [comment]
image

File: visualizer/index.html [hunk 1]
```diff
@@ -37,9 +37,10 @@
 <body>
   <div class="mainHeading">
     <h1 style="margin-top:0"><img src="img/flink-logo.png" width="100" height="100" alt="Flink Logo" align="middle"/>Flink Plan Visualizer
-	    <div style="position:absolute; top:40px; right:110px;">
+	    <div style="position:absolute; top:40px; right:60px;">
 		  <button id="zoomIn" type="button" class="btn btn-default">Zoom In</button>
-		  <button id="zoomOut" type="button" class="btn btn-default">Zoom Out</button>
+		  <button id="zoomOut" type="button" class="btn btn-default">Zoom Out</button>&nbsp;&nbsp;
+          <button id="reset" type="button" class="btn btn-default" onclick="window.location.reload()">Reset</button>
```
Comment (zentol) [inline visualizer/index.html]: the indentation appears to be off.

Comment (yew1eb) [reply visualizer/index.html]: done.

**Comment by zentol** [comment]
Will test this locally and merge it afterwards.

**Pull request closed without merging**